internal/tiltfile/docker.go

```diff
@@ -43,6 +43,31 @@ func (d *dockerImage) ID() model.TargetID {
 	return model.ImageID(d.ref)
 }
 
+type dockerImageBuildType int
+
+const (
+	StaticBuild = iota
+	FastBuild
+	CustomBuild
+	UnknownBuild
```

Comment:
It probably doesn't really matter given how this is currently being used, but I think it's good to put `Unknown` at the top of the iota, so that we're more likely to catch uninitialized values rather than unintentionally plodding along with StaticBuild. I thought I read this in Effective Go, but now I can't find a citation. Anyway, it seems like a good idea.